HEADER.js

```diff
@@ -71,95 +77,25 @@ fabric.SHARED_ATTRIBUTES = [
 ];
 /* _FROM_SVG_END_ */
 
-/**
- * Pixel per Inch as a default value set to 96. Can be changed for more realistic conversion.
- */
-fabric.DPI = 96;
+
 fabric.reNonWord = /[ \n\.,;!\?\-]/;
-fabric.fontPaths = { };
-fabric.iMatrix = iMatrix;
 
 
-/**
- * Pixel limit for cache canvases. 1Mpx , 4Mpx should be fine.
- * @since 1.7.14
- * @type Number
- * @default
- */
-fabric.perfLimitSizeTotal = 2097152;
 
 /**
- * Pixel limit for cache canvases width or height. IE fixes the maximum at 5000
- * @since 1.7.14
- * @type Number
- * @default
+ * @todo move to config when window is exported
  */
-fabric.maxCacheSideLimit = 4096;
-
-/**
- * Lowest pixel limit for cache canvases, set at 256PX
- * @since 1.7.14
- * @type Number
- * @default
- */
-fabric.minCacheSideLimit = 256;
+config.configure({
+  devicePixelRatio: fabric.window.devicePixelRatio ||
+    fabric.window.webkitDevicePixelRatio ||
+    fabric.window.mozDevicePixelRatio ||
```

Comment:
i think we can verify which browser still need those, and we can removed if is something super old.